lib/teracy-dev-essential.rb

```diff
@@ -3,15 +3,15 @@
 require_relative 'teracy-dev-essential/config/host_manager'
 require_relative 'teracy-dev-essential/config/save_mac_address'
 require_relative 'teracy-dev-essential/config/networks'
-require_relative 'teracy-dev-essential/config/rsync_recovery'
+require_relative 'teracy-dev-essential/config/gatling_rsync_recovery'
```

Comment:
I don't think we need to rename this file.

As on Linux with vagrant >= 2.2.3, we'll use vagrant rsync-auto instead of vagrant gatling-rsync-auto on the trigger part of this file, related: #40